### PR TITLE
Revendor gardener/gardener@v1.16.0

### PR DESCRIPTION
**Pull request by prashanth26**
**How to categorize this PR?**

/area dev-productivity
/kind enhancement
/priority normal
/platform aws

**What this PR does / why we need it**:
Revendor gardener/gardener@v1.16.0

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
Would help with testing the latest g/g changes on the extension. 

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/coreos/go-systemd/v22 v22.1.0
 	github.com/frankban/quicktest v1.9.0 // indirect
 	github.com/gardener/etcd-druid v0.3.0
-	github.com/gardener/gardener v1.15.1-0.20210118135118-1c2defb1ce0b
+	github.com/gardener/gardener v1.15.1-0.20210127160255-46acdf6fc0ba
```
Comment (ialidzhikov) [inline go.mod:11]:
```suggestion
	github.com/gardener/gardener v1.16.0
```

Comment (prashanth26) [reply go.mod]: Made the suggested change [here](https://github.com/gardener/gardener-extension-provider-aws/pull/262/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R11) and ran `make revendor`.

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by ialidzhikov — APPROVED** [review]
/lgtm

**Pull request merged**